docs/controls/WebView.md

```diff
@@ -318,9 +306,9 @@ Scripts in the web view content can use **window.external.notify** with a string
 
 You can use the [Settings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.settings) property (of type [WebViewControlSettings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrolsettings) to control whether JavaScript and IndexedDB are enabled. For example, if you use a web view to display strictly static content, you might want to disable JavaScript for best performance.
 
- ## Requirements
+## Requirements
 
-| Device family | .NET 4.7, 10.0.17110.0 or higher |
+| Device family | .NET 4.6.2, 10.0.17110.0 or higher |
```

Comment:
I believe we should we use the now released 17134 build number here for the OS version?

Reply:
@michael-hawker I don't think so: Usage of our control depends on OS version 10.0.17110.0 or greater, and not the April 2018 update. That is, a developer may use our control on a previous insider flight without issue. I've updated the language here to mimic what's already out there on docs.microsoft.com to make that more clear.

Reply:
This is intended to state the minimum version, rather than the target version. Right now the public version is April 2018 Update, but we have insider flights that have improvements. The machine must AT LEAST have 17110, 17134 is better, and 17655 (Insider Skip) is better still